packages/graphql-testing/README.md

```diff
@@ -0,0 +1,94 @@
+# `@shopify/graphql-testing`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Fgraphql-testing.svg)](https://badge.fury.io/js/%40shopify%2Fgraphql-testing.svg)
+
+This utility aid with the following testing scenarios:
+
+1. Testing graphql request with mock data.
+2. Testing the state of your application before / after graphql request(s) resolves.
```

Comment:
`2. Testing the state of your application before/after one or more graphql requests resolve.`